src/Glue.elm

```diff
@@ -71,6 +71,52 @@ glue =
     Glue
 
 
+{-| Create [Glue](#Glue) mapigs between modules.
+child module can be generic TEA app or module that is already doing polymorfic maping to generic `msg` internaly.
+You can also use `Cmd` for sending data from bottom module to upper one if you want to observe child
+as a black box (similary you do in case of DOM events with `Html.Events`).
+
+**Interface**:
+
+    createglue :
+          { modelsetter = \subModel model -> { model | subModel = subModel }
+          , modelgetter = .subModel
+          , init =  ( subModel, Cmd subMsg )
+          , update = subMsg -> subModel -> ( subModel, Cmd msg )
+          , view = subModel -> Html subMsg
+          , subscriptions = subModel -> Sub subMsg
+          , liftmessage = subMsg -> msg
+          }
+        -> Glue model subModel msg subMsg
+
+See [examples](https://github.com/turboMaCk/glue/tree/master/examples) for more informations.
+
+-}
+createglue :
+    { e
+        | init : ( subModel, Cmd subMsg )
+        , liftmessage : subMsg -> msg
+        , modelgetter : model -> subModel
+        , modelsetter : subModel -> model -> model
```

Comment:
Not a bad but you're right they are slightly longer. Also I think convention is to use `camelCase` so it will become `subModelAccessor` and `subModelUpdater` but that's just detail. Will think about this as well.